src/Components/App/App.js

```diff
@@ -32,86 +32,104 @@ class App extends Component {
 
   getActivityData = async (event) => {
     let promise;
-    try{
-      if(event.target.id === 'movies' ) {
-          promise = await getAllMovies();
+    try {
+      if (event.target.id === 'movies') {
+        promise = await getAllMovies();
       }
-      if(event.target.id === 'boardGames' ) {
+      if (event.target.id === 'boardGames') {
         promise = await getAllBoardGames();
       }
-      if(event.target.id === 'cardGames' ) {
+      if (event.target.id === 'cardGames') {
         promise = await getAllCardGames();
       }
-      if(event.target.id === 'music' ) {
-          promise = await getAllMusic();
+      if (event.target.id === 'music') {
+        promise = await getAllMusic();
       }
-      if(event.target.id === 'podcasts' ) {
+      if (event.target.id === 'podcasts') {
         promise = await getAllPodcasts();
       }
-      this.setState({ [event.target.id]: promise})
-      } catch (error){
-        console.log(error)
-      }
+      this.setState({[event.target.id]: promise})
+    } catch (error) {
+      console.log(error)
+    }
   }
-  
+
   updateActivityAnswers = (event) => {
     if (event.target.id !== 'default') {
-      this.setState({ [event.target.id]:  [...this.state[event.target.id], event.target.textContent]})
+      this.setState({[event.target.id]: [...this.state[event.target.id], event.target.textContent]})
     }
   }
 
   setActivities = (activities) => {
     this.setState({activities: [...activities]})
   }
 
-  render() {
+  determineRandomActivity = () => {
+    console.log('yeet')
+    // redirect to temporary loading page while the below logic is run
+    // what activities did the user select?
+    // fetch all activities and place somewhere to be filtered through, here? Result component?
+    // if Result component we will also have to pass the answers of the questions down
+    // filter through All results from fetch with specific conditions based on selected answers
+    // store these further filtered results in somewhere where we can access them more than once, state?
+    // randomly choose one of these further filtered activities, maybe remove this from state at that time? 
+    // render component that will display the previewcard
+    // let user select to see more info or skip
+    // if user selects skip, get another random activity from set which should no longer include the one they skipped if we removed it
```

Comment:
I like your thought process here about removing the skipped activity from state; would lead to better use experience in the event that skipped activity were to show up twice.